src/utils/apis/LoginAPI.js

```diff
@@ -36,6 +42,18 @@ export const loginAPI = {
                 resolve({text: "ok", ok: true})
             }, 2000)
         })
-    }
+    },
 
+    saveTokenFromResponse: function(response) {
+        localStorage.setItem(loginAPI.authTokenStorageKey, response.headers.get(loginAPI.authTokenKey))
+        localStorage.setItem(loginAPI.refreshTokenStorageKey, response.headers.get(loginAPI.refreshTokenKey))
```

Comment:
Jak w końcu robimy z tym refreshem? Będzie, nie będzie? Jak będzie to w tym tasku powinieneś jeszcze automatyczne odświeżanie zrobić.